### PR TITLE
[Finishes #101464522] use small footer for applab

### PR DESCRIPTION
**Pull request by Bjvanminnen**
image

image

### CODE REVIEW

File: dashboard/app/assets/stylesheets/application.css.scss [hunk 1]
```diff
@@ -370,10 +370,12 @@ $small-footer-standard-width: 400px;
   z-index: 1;
```
Comment (islemaster) [inline dashboard/app/assets/stylesheets/application.css.scss]: Do you need to modify the z-index for `#more-menu` as well?

**Comment by islemaster** [comment]
Is it a problem that this footer now appears above the Admin box?

**Comment by islemaster** [comment]
When integrating the small footer into NetSim I had to make sure the left column stopped above the footer, so any longer content would be full visible or scroll correctly.  For now, it looks like there is no such left column content in AppLab.  Is that something we should anticipate now?

File: dashboard/app/assets/stylesheets/application.css.scss (hunk 1 above)
Comment (Bjvanminnen) [inline dashboard/app/assets/stylesheets/application.css.scss]: It doesnt currently overlap with anything, but it's probably good to bump this too for consistency

**Comment by Bjvanminnen** [comment]
The admin/footer overlap is kind of annoying, but orthogonal to this change (we're also going to get the small footer on all blockly levels in the new future, so we may find ourselves annoyed enough to move the admin box soon).

**Comment by Bjvanminnen** [comment]
Regarding your comment about the left column, are you referring to things like instructions? I validated that those will scroll as we resize the window:

image

**Comment by islemaster** [comment]
Oh yeah, the left column in this case may be using common code and position above the footer already.  It would be helpful if a scroll bar showed up in that screenshot :grin:.

More alarming is how the footer is overlapping with the debug area in the screenshot you posted.  Resize debounce issue, or was the width persistently wrong?

**Comment by Bjvanminnen** [comment]
I think scrollbar not showing up is a mac thing (it disappears when not actively scrolling)

Footer not resizing looks to be a legit issue. Doesn't seem to repro on blockly levels. Will investigate what's different about applab

**Comment by Bjvanminnen** [comment]
Actually, admin overlapping footer becomes more annoying after we moving footer in front. I'll probably just bump the admin box up to be above the footer

**Comment by Bjvanminnen** [comment]
Turns out I only repro the footer not resizing when I open Chrome console. For regular resizes (i.e. dragging corner of window) it appears to do the right thing

**Comment by islemaster** [comment]
LGTM :+1: Thanks for investigating discussed edge cases.  I also repro the console doesn't cause appropriate resize event thing, I don't think we need to worry about it.

**Pull request merged**